e2e/node/resource_node.go

```diff
@@ -431,6 +431,19 @@ func resourceUpdateNode(ctx context.Context, d *schema.ResourceData, m interface
 		}
 	}
 
+	if d.HasChange("plan") {
+		prevPlan, currPlan := d.GetChange("plan")
+		log.Printf("[INFO] prevPlan %s, currPlan %s", prevPlan.(string), currPlan.(string))
+		d.Set("plan", prevPlan)
+		return diag.Errorf("Plan cannot be updated once you create the node.")
```

Comment:
plan upgrade is possible currently we are not providing this functionality kindly update the message accordingly